tree/30_generic_methods/logger_rudder.cf

```diff
@@ -33,7 +33,7 @@ bundle agent logger_rudder(message, class_prefix)
       "expected_reports_temp"   string => "${expected_reports_source}.tmp";
       "expected_reports_file"   string => "${expected_reports_source}.res";
 
-    (!final_resfile_exists.logger_rudder_temp_resfile_reached|logger_rudder_temp_resfile_repaired)::
+    pass2.(!final_resfile_exists.logger_rudder_temp_resfile_reached|logger_rudder_temp_resfile_repaired)::
```

Comment:
ha, of course. 
final_resfile_exists is not yet defined, but all other classes are already defined (classes as promise outcome remain for the rest of the run)
good catch